app/javascript/components/ansible-playbook-edit-catalog-form/helper.js

```diff
@@ -0,0 +1,315 @@
+import { useFieldApi, useFormApi } from '@@ddf';
+import React from 'react';
+import { Button, TextInput, Checkbox, Select, SelectItem } from 'carbon-components-react';
+import { TrashCan32 } from '@carbon/icons-react';
+import { TreeViewRedux } from '../tree-view';
+
+const convertObject = (inputObj) => {
+  const outputArray = [];
+  for (const [key, value] of Object.entries(inputObj)) {
+    outputArray.push({ key, value: value.default });
+  }
+  return outputArray;
+};
+
+const convertArrayToObject = (arr) => arr.reduce((acc, curr) => {
+  acc[curr.key] = { 'default': curr.value };
+  return acc;
+}, {});
+
+const getTenantId = (key) => {
+  if (key.startsWith('tn')) {
+    return key.split('-')[1];
+  }
+};
+
+const getSortedHash = function (inputHash) {
```

Comment:
`const getSortedHash = (inputHash) => {
`

Reply:
made the changes.